README.md

```diff
@@ -141,6 +141,12 @@ Study](https://www.citusdata.com/solutions/case-studies/agari-case-study)
 funnel, segmentation, and cohort queries across billions of users
 and tens of billions of events. [Watch
 Video](https://www.youtube.com/watch?v=NVl9_6J1G60&list=PLixnExCn6lRpP10ZlpJwx6AuU3XIgNWpL)
+* [Pex](https://pex.com/) uses Citus to implement very high volume data creation and search database for audio/video data on a 20+ node cluster on Google Cloud [Case Study](https://www.citusdata.com/customers/pex)
```

Comment:
Two small edit proposals. Please evaluate and incorporate as you see fit.

1. "Pex uses Citus to ingest 30B data points per day and analyze that data in real-time. They use a 20+ node cluster on Google Cloud."
2. "Algolia uses Citus to provide real-time analytics for over 1B searches per day. For faster insights, they also use TopN and HLL extensions."

Reply:
This is my first ever github comment. :-) Here are a few small things:

1. please change most, but not all, references to "Case Study" to "Customer Story"
2. at the bottom, please change to: "You can read more customer stories about how users employ Citus to scale Postgres for both multi-tenant SaaS applications as well as real-time analytics dashboards.
3. I've sent a note off to Rasty from Pex to confirm the size of his cluster. I think he upgraded since we wrote the story.

Reply:
Rasty responds quickly. Pex is still using a 20-node cluster.

Reply:
@tgcmn1 Can we not use "Customer" here. "User" seems to be a better option